src/core/status_cache.zig

```diff
@@ -186,9 +187,71 @@ pub const StatusCache = struct {
 
 pub const Ancestors = struct {
     // agave uses a "RollingBitField" which seems to be just an optimisation for a set
-    ancestors: HashMap(Slot, void) = .{},
+    ancestors: Map = .{},
+
+    pub const @"!bincode-config:ancestors" = mapConfig();
+    pub const Map = HashMap(Slot, void);
+
+    fn mapConfig() bincode.FieldConfig(Map) {
+        // [agave] https://github.com/anza-xyz/agave/blob/85b647f149b0f08d30e6cd6081d50e87b3cc6c20/accounts-db/src/ancestors.rs#L54
+        // For some reason, agave serializes Ancestors as HashMap(slot, usize). But deserializing
+        // ignores the usize, and serializing just uses the value 0. So we need to serialize void
+        // as if it's 0, and deserialize 0 as if it's void.
+        const VoidCfg = struct {
+            fn deserialize(alloc: std.mem.Allocator, reader: anytype, params: bincode.Params) !void {
+                _ = try bincode.read(alloc, usize, reader, params);
+            }
+            fn serialize(writer: anytype, data: anytype, params: bincode.Params) !void {
+                _ = data;
+                try bincode.write(writer, @as(usize, 0), params);
+            }
+        };
```

Comment:
Could move this namespace outside of the function, and can just inline the return value into where `mapConfig()` is, no point in having this wrapper function imo.

Reply:
e27778a0f97665fe9e2b39185155e7a4a28a80e7